build.sbt

```diff
@@ -0,0 +1,78 @@
+ThisBuild / organization := "za.co.absa"
```

Comment:
Why `sbt` when we use `pom` in all other projects?

Reply:
- everything is in scala or SBT DSL, no stupid xml, ergo cleaner
- easier dependency management
- easier cross-compilation
- considered "native"
- Cobrix moved to SBT
- I just wanted to